pkg/nfd-master/nfd-master.go

```diff
@@ -576,39 +572,38 @@ func addAnnotations(n *api.Node, annotations map[string]string) {
 	}
 }
 
-
-func updateMap(input map[string]int32)map[string]int{
+func updateMap(input map[string]int32) map[string]int {
 	ret := make(map[string]int)
 
-	for str, data := range input{
-		ret[str]=int(data)
+	for str, data := range input {
+		ret[str] = int(data)
 	}
 	return ret
 }
 
-func modifyCRD(topoUpdaterZones map[string]*topologypb.Zone)map[string]v1alpha1.Zone{
+func modifyCRD(topoUpdaterZones map[string]*topologypb.Zone) map[string]v1alpha1.Zone {
 
 	zones := make(map[string]v1alpha1.Zone)
-	for zoneName, zone := range topoUpdaterZones{
-		resInfo := make(	map[string]v1alpha1.ResourceInfo)
-		for resourceName, info := range zone.Resources{
+	for zoneName, zone := range topoUpdaterZones {
+		resInfo := make(map[string]v1alpha1.ResourceInfo)
+		for resourceName, info := range zone.Resources {
 			resInfo[resourceName] = v1alpha1.ResourceInfo{
-					Allocatable: info.Allocatable,
-					Capacity: info.Capacity,
-				}
+				Allocatable: info.Allocatable,
+				Capacity:    info.Capacity,
+			}
 		}
 
-		zones[zoneName]= v1alpha1.Zone{
-				Type: zone.Type,
-				Parent: zone.Parent,
-				// Costs: updateMap(zone.Costs),
-				// Attributes: updateMap(zone.Attributes),
-		/*
-		Costs: zone.Costs,
-		Attributes: zone.Attributes,
+		zones[zoneName] = v1alpha1.Zone{
+			Type:   zone.Type,
+			Parent: zone.Parent,
+			// Costs: updateMap(zone.Costs),
```

Comment:
The zone cost is not being updated here which means that the CRD is not created with the cost. Just commenting this would make it work.

Reply:
right! let me fix this